usaspending_api/etl/elasticsearch_loader_helpers/fetching_data.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import os
+import subprocess
+
+from django.conf import settings
+from time import perf_counter, sleep
+
+from usaspending_api.common.csv_helpers import count_rows_in_delimited_file
+from usaspending_api.common.helpers.sql_helpers import get_database_dsn_string
+from usaspending_api.etl.elasticsearch_loader_helpers.utilities import format_log, execute_sql_statement, DataJob
+
+logger = logging.getLogger("script")
+
+COUNT_FY_SQL = """
+SELECT COUNT(*) AS count
+FROM {view}
+WHERE {fiscal_year_field}={fy} AND update_date >= '{update_date}'
+"""
+
+COUNT_SQL = """
+SELECT COUNT(*) AS count
+FROM {view}
+WHERE update_date >= '{update_date}'
+"""
+
+COPY_SQL = """"COPY (
+    SELECT *
+    FROM {view}
+    WHERE {fiscal_year_field}={fy} AND update_date >= '{update_date}'
+) TO STDOUT DELIMITER ',' CSV HEADER" > '{filename}'
+"""
+
+# ==============================================================================
+# Other Globals
+# ==============================================================================
+
+AWARD_DESC_CATEGORIES = {
+    "loans": "loans",
+    "grant": "grants",
+    "insurance": "other",
```

Comment:
Is that right? `insurance` goes under `other` right now?

Reply:
For now. There is a desire to split it into it's own category in the future